src/main/java/com/twcable/grabbit/cli/Utils.java

```diff
@@ -13,26 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.twcable.grabbit.cli
+package com.twcable.grabbit.cli;
 
-class ThreadedStreamProcessor extends Thread {
-    InputStream inputStream
-    def output = []
+class Utils {
 
-
-    ThreadedStreamProcessor(InputStream inputStream) {
-        this.inputStream = inputStream
+    /**
+     * Remove checked-ness from the exception. The same exception is still thrown (checked or unchecked), but this
+     * removes the compiler's checks.
+     */
+    @SuppressWarnings("RedundantTypeArguments")
+    public static <T extends Throwable> void throwSoft(T exp) {
+        throw Utils.<RuntimeException>uncheck(exp);
```

Comment:
Besides the obvious getting rid of checked exceptions noisiness, it's real value comes from the fact that it makes using the standard functional interfaces much, much cleaner. They don't declare a `throws` signature so you need to handle things inside the lambda, which is really, *really* ugly. This way you're not swallowing exceptions, but also not having to do the nastiness inside lambdas.